lib/rest-api.php

```diff
@@ -460,3 +460,33 @@ function gutenberg_silence_rest_errors() {
 	}
 
 }
+
+/**
+ * Include additional labels for registered post types
+ *
+ * @return array Arguments supplied to register_post_type()
+ */
+function gutenberg_filter_post_type_labels( $args, $post_type ) {
+	if ( is_post_type_hierarchical( $post_type ) ) {
+		$labels = array(
+			'published'           => __( 'Page published.' ),
+			'published_privately' => __( 'Page published privately.' ),
+			'reverted_to_draft'   => __( 'Page reverted to draft.' ),
+			'scheduled'           => __( 'Page scheduled.' ),
+			'updated'             => __( 'Page updated.' ),
+			'view'                => __( 'View page.' ),
```

Comment:
For `'view'` specifically, it looks like we already have a `'view_item'` label we should reuse.

For the others, I think the keys need to be a bit more specific. What about `'published_item'`, `'published_item_privately'`, etc.?